packages/web/pages/apps/index.tsx

```diff
@@ -0,0 +1,163 @@
+import Fuse from "fuse.js";
+import React from "react";
+import { useTranslation } from "react-multi-lang";
+
+import { Button } from "~/components/buttons";
+import { HeroCard } from "~/components/cards";
+import AppDisplayCard from "~/components/cards/app-display-card";
+import { SearchBox } from "~/components/input";
+import { EventName } from "~/config";
+import { useAmplitudeAnalytics } from "~/hooks";
+
+type appDataType = {
+  title: string;
+  subtitle: string;
+  thumbnail_image_URL: string;
+  external_URL: string;
+  twitter_URL?: string;
+  github_URL?: string;
+  medium_URL?: string;
+  featured?: boolean;
+};
+
+type App = {
+  title: string;
+  subtitle: string;
+  external_URL: string;
+  thumbnail_image_URL: string;
+  hero_image_URL: string;
+  twitter_URL?: string;
+  medium_URL?: string;
+  github_URL?: string;
+  featured?: boolean;
+};
+
+type AppStoreProps = {
+  apps: {
+    applications: App[];
+  };
+};
+
+export const AppStore: React.FC<AppStoreProps> = ({ apps }) => {
+  const [searchValue, setSearchValue] = React.useState<string>("");
+  const [fuzzySearchResults, setFuzzySearchResults] = React.useState<
+    appDataType[]
+  >([]);
+
+  const { applications } = apps;
+
+  const t = useTranslation();
+  const { logEvent } = useAmplitudeAnalytics();
+
+  let featuredApp = applications?.find((app) => app.featured === true);
+  const nonFeaturedApps = applications?.filter((app) => !app.featured);
+
+  featuredApp = featuredApp ? featuredApp : nonFeaturedApps[0];
+
+  const options = {
+    keys: ["title"],
+  };
+  const fuse = new Fuse(applications, options);
+
+  const handleApplyClick = () => {
+    logEvent([EventName.AppStore.applyClicked]);
+    const url = "https://cosmos-ecosystem.webflow.io/submit";
+    window.open(url, "_blank");
+  };
+
+  const handleSearchInput = (value: string) => {
+    const searchResults = fuse.search(value);
+    const appDataResults: appDataType[] = searchResults.map(
+      (result) => result.item as appDataType
+    );
+
+    setSearchValue(value);
+    setFuzzySearchResults(appDataResults);
+  };
+
+  const iterableData = searchValue ? fuzzySearchResults : nonFeaturedApps;
+
+  return (
+    <main className="m-auto max-w-container bg-osmoverse-900 py-3 md:px-3">
+      <div className="flex justify-between pl-6">
+        <div style={{ flexBasis: "55%" }}>
+          <h4 className="pb-2 text-h4 font-h4 text-wosmongton-100">
```

Comment:
FYI in globals.css we apply some of these tailwind font/text styles to the h4 tag automatically:
https://github.com/osmosis-labs/osmosis-frontend/blob/115004828f3295d17130a0141688e1a4221783b8/packages/web/styles/globals.css#L33

For the most part, you shouldn't need to apply both the text- and the font- tag as they're available as a single class in the link above. Saves text.